### DISCUSSION TITLE
irssi

### DISCUSSION THREAD

**Discussion by aquifolly** [catppuccin/catppuccin#2348] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

examples here: https://irssi-import.github.io/themes/

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by backwardspy** [comment]
hey, thanks for raising this! i've spent some time on it, and i think there's very little for us to port here.

irssi is well-behaved in that it uses your terminal colours, and themes exist solely to set formatting and where exactly those colours go. as such you can use a lot of the themes on the page you linked with your terminal set to catppuccin and you'll get a decent result.

as an example; the default theme _almost_ looks good enough as it is, but the bright blue statusbar was not readable so i changed that from blue to black:

```diff
253,254c253,254
<   sb_background = "%4%w";
<   window_border = "%4%w";
---
>   sb_background = "%0%w";
>   window_border = "%0%w";
```

result:
![preview](https://github.com/catppuccin/catppuccin/assets/289746/26fbc98c-cb2e-4c18-8f23-9a2cfe32654a)

given how well this works, i don't see a need at this stage for us to create a port. if you think there is still value to be had from an official irssi port then please let me know!
